src/components/TricotsStart.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from "react";
+import {Helmet} from "react-helmet";
+import {connect} from "react-redux";
+import * as maillotActions from "../actions/maillotActions";
+import * as riderStageConnectionsActions from "../actions/riderStageConnectionsActions";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'riderStageConnectionsActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)